services/rollout-service/pkg/versions/versions.go

```diff
@@ -98,27 +98,21 @@ func (v *versionClient) GetVersion(ctx context.Context, revision, environment, a
 }
 
 // Tries getting the version from cache
-func (v *versionClient) tryGetVersion(ctx context.Context, revision, environment, application string) (*VersionInfo, error) {
-	var overview *api.GetOverviewResponse
-	entry, ok := v.cache.Get(revision)
+func (v *versionClient) tryGetVersion(environment, application string) (*VersionInfo, error) {
```

Comment:
Previously, we hand't deleted the Environment Application information from the overview, so this function still worked. As we deleted it, now we were forced to rewrite this function